source/elements/oneTBB/source/algorithms/functions/collaborative_once_flag_cls.rst

```diff
@@ -0,0 +1,33 @@
+.. SPDX-FileCopyrightText: 2019-2021 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+=======================
+collaborative_once_flag
+=======================
+**[algorithms.collaborative_call_once.collaborative_once_flag]**
+
+Helper structure for ``collaborative_call_once``.
+
+``collaborative_once_flag`` is passed to ``collaborative_call_once`` as the first argument and serves to resolve
+which of the multiple callers under this flag will be the *Winner* and which will collaborate.
```

Comment:
```suggestion
Special class that ``collaborative_call_once`` uses to perform a call only once.
```
Why it is described in two places. Maybe remove it from collaborative_call_once_func.rst?

Reply:
My idea was to leave some short description in collaborative_call_once_func.rst but provide more details in  collaborative_once_flag_cls.rst